ScreenBoom/ScreenBoom/ViewController/PlayEventViewController.swift

```diff
@@ -147,30 +153,26 @@ class PlayEventViewController: BaseViewController, PlayEventViewModelSourceObser
                               width: self.view.bounds.width,
                               height: self.view.bounds.height)
     }
+    
   }
   
   func setupSideMenu() {
-    // create rightMenuView to show options
-    let rightMenuView = RightMenuView(frame: CGRect(x: self.view.frame.maxX - 10,
-                                                    y: 0,
-                                                    width: 80,
-                                                    height: self.view.frame.height))
-    self.view.addSubview(rightMenuView)
-    rightMenuView.configureWith(eventName: self.event.eventName, eventCode: self.event.eventCode)
-    rightMenuView.eventNameAndCodeButtonLabel.setTitle("Title: \(self.event.eventName) \n Code: \(self.event.eventCode)", for: .normal)
-    rightMenuView.shareImage.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(handleTapGuestureOnShareImage)))
-    rightMenuView.pauseImage.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(handleTapGestureOnPauseImage)))
-    rightMenuView.playImage.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(handleTapGestureRecognizerOnPlayImage)))
-    rightMenuView.deleteImage.addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(handleTapGestureRecognizerOnDeleteImage)))
-    rightMenuView.editButton.addTarget(self, action: #selector(handleEditButtonPressed(_:)), for: .touchUpInside)
-    // show this options for event owner only (Edit, play, pause and delete)
-    if let userID = UserDefaults.standard.object(forKey: userDefaultKeyNames.userIDKey) as? String,
-      userID == self.event.userID {
-      print("isOwner")
-      [rightMenuView.editButton, rightMenuView.deleteImage, rightMenuView.pauseImage, rightMenuView.playImage].forEach{$0.isHidden = false}
-    }
-    self.view.bringSubview(toFront: rightMenuView)
-    self.rightMenuView = rightMenuView
+    
+    let rightMenu = RightMenuView()
+    
+    self.view.addSubview(rightMenu)
+    rightMenu.frame = CGRect(x: self.view.frame.maxX - 10,
+                             y: 0,
+                             width: 80,
+                             height: self.view.frame.height)
+    
+    rightMenu.configureWith(event: self.event, eventCode: self.event.eventCode)
+   
+    self.view.bringSubview(toFront: rightMenu)
+    self.rightMenuView = rightMenu
+    
+    // Delegates
+    self.rightMenuView?.sideMenuDelegate = self
```

Comment:
for all of these addEventToUserEvents.. and the other ones, can we clean them up a little ? just the alignment and stuff so its easier to read